src/components/Subheader/Subheader.js

```diff
@@ -0,0 +1,51 @@
+import React, { useState } from 'react';
+import useStyles from '../Subheader/Subheader.styles.js';
+import CloseIcon from '@material-ui/icons/Close';
+import PropTypes from 'prop-types';
+import useMediaQuery from '@material-ui/core/useMediaQuery';
+
+export default function Subheader(props) {
+  const { variant, mainLabel, subLabel, image, testClick } = props;
```

Comment:
deconstruct at method signature level..

`export default function Subheader({ variant, mainLabel, subLabel, image, testClick }) { .......`

Reply:
Updated props deconstruction  in commit - 13861f493a78d9f22c558ee5adf75a647dcc966c